.gitignore

```diff
@@ -1,2 +1,3 @@
 .DS_Store
 *.class
+classes/images
```

Comment:
Just to help cut down on random files showing up potentially for adding repository